layers/+lang/python/config.el

```diff
@@ -24,3 +24,6 @@
 
 (defvar python-fill-column 79
   "Fill column value for python buffers")
+
+(defvar python-sort-imports-on-save nil
```

Comment:
I think `python-enable-sort-imports-on-save` is more consistent with other layer variables